qemu/tests/cfg/hotplug_mem_negative.cfg

```diff
@@ -0,0 +1,43 @@
+- hotplug_mem_negative:
+    type = hotplug_mem_negative
+    mem_fixed = 4096
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 32G
+    guest_numa_nodes = "node0"
+    ppc64,ppc64le:
+        threshold = 0.15
+    policy_mem = default
+    target_mems = "plug1"
+    set_addr = no
+    variants:
+        - overcommit_host_mem:
+            size_mem = 1000G
+            keywords = "Cannot allocate memory"
+        - min_mem_size:
+            size_mem = 0G
+            keywords = "memory-backend-ram.size\' doesn\'t take value \'0\'"
+        - invalid_backend_ram_option:
+            policy_mem = bind
+            host-nodes = 3
```

Comment:
Please choose the node according to host numa topolgy.

Reply:
corrected